go/runtime/txpool/txpool.go

```diff
@@ -83,6 +83,11 @@ type TransactionPool interface {
 	// ClearProposedBatch clears the proposal queue.
 	ClearProposedBatch()
 
+	// RejectTxs indicates that given transaction hashes have been rejected during block processing.
+	// Queues that can remove those transactions will do so. Additionally, these transactions will
+	// be removed from the already seen cache as they can potentially become valid in the future.
```

Comment:
The final two sentences might be too detailed for an interface as they delve into implementation specifics. But so are some other comments.